ci/docker/install/ubuntu_adduser.sh

```diff
@@ -34,4 +34,5 @@ then
     mkdir /work/mxnet    
     mkdir /work/build
     chown -R jenkins_slave /work/
+    chown -R jenkins_slave /usr/local/bin
```

Comment:
I think this is hacky.

Reply:
The problem is that the entire Docker container is owned by root by default. Since we run in low-privilege mode inside our container (opposed to running everything as root), we have to whitelist certain directories (/work and /usr/local/bin) that the process is allowed to write. This is basically a necessary evil of the least-priviledge-policy. 

An alternative would be to set these softlinks directly during the ccache installation. The problem there would then be that CCache would **always** be activated and we would not be able to turn it on/off during runtime. I wouldn't mind about that, but I didn't want to change the behaviour. What do you think?

Reply:
What requires permissions to write in /usr/local/bin ?

Reply:
The symbolic link I'm creating. It's super hacky, but some buildsystems seem to directly access /usr/local/bin in order to access the compiler binary. They ignore any custom setting, so I'm using the super brutal way to just rewrite that entry by overriding it with a symlink